tests/integration/kubernetes/confidential/confidential_common.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bats
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source "../../tests_common.bash"
+
+setup_unencrypted_confidential_pod() {
+    local pod_name="unencrypted-confidential"
+    local IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+    local UNENCRYPTED_IMAGE_URL="${IMAGE_REPO}:unencrypted"
+    local TEST_DIR="$(mktemp -d /tmp/test-kata-confidential.XXXXXXXX)"
+    export SSH_KEY_FILE="${TEST_DIR}/container-ssh-key"
+
+   	k8s_generate_service_yaml "${pod_config_dir}/${pod_name}.yaml" "${UNENCRYPTED_IMAGE_URL}"
+
+    if [ "${KATA_HYPERVISOR}" == "qemu-sev" ]; then
+        # Pull image
+        docker pull "${UNENCRYPTED_IMAGE_URL}"
+        
+        # Get ssh key from docker image label and save to file
+        docker inspect ${image_url} \
+            | jq -r '.[0].Config.Labels.ssh_key' \
+            | sed "s|\(-----BEGIN OPENSSH PRIVATE KEY-----\)|\1\n|g" \
+            | sed "s|\(-----END OPENSSH PRIVATE KEY-----\)|\n\1|g" \
+            > "${SSH_KEY_FILE}"
+
+        # Set permissions on private key file
+        chmod 600 "${SSH_KEY_FILE}"
+
+        if [ "${KATA_HYPERVISOR}" == "qemu-sev" ]; then
```

Comment:
This condition is unnecessary

Reply:
Yes, that was the residual code when I tried splitting it into 2 commits. I have fixed that now. Thanks.